provisioning_client/CMakeLists.txt

```diff
@@ -196,7 +196,7 @@ target_link_libraries(prov_auth_client hsm_security_client)
 set(provisioning_libs ${provisioning_libs} prov_auth_client)
 set(provisioning_headers ${provisioning_headers} ${AUTH_CLIENT_H_FILES})
 
-if(${use_prov_client})
+if(${use_prov_client} OR (${use_prov_client_core} AND ${run_e2e_tests}))
```

Comment:
>AND ${run_e2e_tests} [](start = 50, length = 20)

I'm pretty sure you don't want run_e2e_tests here.  You want some of this stuff done even if run_e2e_test is OFF>

Reply:
discussed offline.  It's ok for now to unblock as we do not want to bring in extras for the case of building for Edge.  The caveat is that we are bringing in more bits for e2e tests only meaning that the bits that we are testing vs in production "COULD" be different.  This is not ideal and we are going to create a backlog item to track the work for investigation/fixing it.